### PR TITLE
Fixes to list tabs.

### PR DESCRIPTION
**Pull request by skurvish**
Added dropdowns as a tab choice.

### CODE REVIEW

**Comment by Sophist-UK** [comment]
I am not able to comment either way from reading the code diff as there are too many changes to be able to make sense of it.

**Comment by skurvish** [comment]
Take a look at the merged function without the diffs, it will makes more sense.

File: components/com_fabrik/models/list.php [hunk 1]
```diff
@@ -11984,6 +11985,17 @@ private function getTabCategories()
 		JDEBUG ? $profiler->mark('after fabrik list tabs query run') : null;
 		FabrikHelperHTML::debug($counts, 'list getTabCategories counts: ' . $table->label);
 
+		if ($isDropdown) {
+			/* Convert element values to names */
+			foreach ($counts as $key => $count) {
+				$counts[$key][0] = $elementModel->getLabelForValue($counts[$key][0]);
+			}
+			/* Now sort it by these labels */
+			usort($counts, function($a, $b) {
```
Comment (Sophist-UK) [inline components/com_fabrik/models/list.php]: Unclear why you are using `usort` rather than simpler `sort`?

Comment (skurvish) [reply components/com_fabrik/models/list.php]: multidimensional array.

Comment (Sophist-UK) [reply components/com_fabrik/models/list.php]: array_multisort?

**Comment by Sophist-UK** [comment]
Still unable to test it, but the concept (as discussed with @skurvish on skype) seems good.

**Pull request merged**